dvc/command/cache.py

```diff
@@ -7,6 +7,9 @@
 
 class CmdCacheDir(CmdConfig):
     def run(self):
+        if not self.args.value:
+            print(self.config["cache"]["dir"])
+            return 0
```

Comment:
We have a similar logic in `dvc config`: https://github.com/iterative/dvc/blob/738dfd49cf403412b018677e62b0b68e85c80886/dvc/command/config.py#L19 , but it also checks for --unset and uses logger.info  (our current convention). Maybe we could do the same here?

Reply:
> We have a similar logic in `dvc config`:


>  , but it also checks for --unset and uses logger.info (our current convention). Seems like we could do the same here.

done.